### PR TITLE
GDB-6858 Returned previous behaviour, but added setting to do not rel…

### PR DESCRIPTION
**Pull request by sava-savov-ontotext**
…oad locations in repository create/edit view.

### CODE REVIEW

**Review by yordanalexandrov — APPROVED** [review]

File: src/js/angular/repositories/controllers.js [hunk 1]
```diff
@@ -772,9 +790,12 @@ function EditRepositoryCtrl($scope, $routeParams, toastr, $repositories, $locati
                     $scope.repositoryInfo.saveId = $scope.saveRepoId;
                     $scope.loader = false;
                 })
-                .error(function (data, status) {
+                .catch(function (response) {
+                    const data = response.data;
+                    const status = response.status;
                     if (status === 404 && $routeParams.repositoryId !== 'system') {
-                        toastr.error('Repo with name ' + '<b>' + $routeParams.repositoryId + '</b>' + ' doesn\'t exists', 'Error', {allowHtml: true});
+                        toastr.error('Repo with name ' + '<b>' + $routeParams.repositoryId + '</b>' + ' doesn\'t exists', 'Error',
```
Comment (svilenvelikov) [inline src/js/angular/repositories/controllers.js:797]: I know it's unrelated with the fix in this MR, but can you also i18n this label

**Review by svilenvelikov — APPROVED** [review]

**Review by yordanalexandrov — APPROVED** [review]

File: src/js/angular/repositories/controllers.js (hunk 1 above)
Comment (yordanalexandrov) [reply src/js/angular/repositories/controllers.js]: Added the label and also refactored the error handling a bit.

**Comment by yordanalexandrov** [comment]
Jenkins retry build

File: test-cypress/package.json [hunk 2]
```diff
@@ -4,7 +4,8 @@
     "description": "Cypress tests for GraphDB workbench",
     "scripts": {
         "start": "cypress open",
-        "test": "cypress run"
+        "test": "cypress run",
+        "test:partial": "cypress run --spec \"integration/repository/**\""
```
Comment (svilenvelikov) [inline test-cypress/package.json:8]: Do you intend to leave this? I like it and if you parametrize it it will be great :)

**Review by svilenvelikov — APPROVED** [review]

File: test-cypress/package.json (hunk 2 above)
Comment (yordanalexandrov) [reply test-cypress/package.json:8]: Not with intention, but since you like it I will leave it :)

**Pull request merged**